src/test/scala/ai/privado/languageEngine/java/passes/config/PropertiesFilePassTest.scala

```diff
@@ -51,41 +52,49 @@ class AnnotationTests extends PropertiesFilePassTestBase(".properties") {
       |
       |class Foo {
       |
+      |private static String loggerUrl;
+      |
       |@Value("${slack.base.url}")
       |private static final String slackWebHookURL;
       |
       |public AuthenticationService(UserRepository userr, SessionsR sesr, ModelMapper mapper,
       |			ObjectMapper objectMapper, @Qualifier("ApiCaller") ExecutorService apiExecutor, SlackStub slackStub,
       |			SendGridStub sgStub, @Value("${internal.logger.api.base}") String loggerBaseURL) {
       |   }
+      |
+      |@Value("${internal.logger.api.base}")
+      |public void setLoggerUrl( String pLoggerUrl )
+      |{
+      |        loggerUrl = pLoggerUrl;
+      |}
       |}
       |""".stripMargin
 
   "ConfigFilePass" should {
     "connect annotated parameter to property" in {
       val anno: List[AstNode] = cpg.property.usedAt.l
-      anno.length shouldBe 2
+      anno.length shouldBe 3
 
-      anno.foreach(element => {
-        element.label match {
-          case "METHOD_PARAMETER_IN" =>
-            val List(param: MethodParameterIn) = element.toList
-            param.name shouldBe "loggerBaseURL"
-          case "MEMBER" =>
-            element.code shouldBe "java.lang.String slackWebHookURL"
-          case _ => s"Unknown label ${element.label}. Test failed"
-        }
-      })
+      anno.code.l shouldBe List(
+        "@Value(\"${internal.logger.api.base}\") String loggerBaseURL",
+        "java.lang.String loggerUrl",
+        "java.lang.String slackWebHookURL"
+      )
     }
 
     "connect property to annotated parameter" in {
-      cpg.property.usedAt.originalProperty.l.length shouldBe 2
-      cpg.property.usedAt.originalProperty.name.l shouldBe List("internal.logger.api.base", "slack.base.url")
-      cpg.property.usedAt.originalProperty.value.l shouldBe List(
+      cpg.property.usedAt.originalProperty.l.length shouldBe 3
+      cpg.property.usedAt.originalProperty.name.toSet.l shouldBe List("internal.logger.api.base", "slack.base.url")
+      cpg.property.usedAt.originalProperty.value.toSet.l shouldBe List(
         "https://logger.privado.ai/",
         "https://hooks.slack.com/services/some/leaking/url"
       )
     }
+
+    "connect the referenced member to the original property denoted by the annotated method" in {
+      cpg.member("loggerUrl").originalProperty.l.name.head shouldBe "internal.logger.api.base"
```

Comment:
Instead of checking on head, let's check on the entire list and also verify that there is only 1 original property for this member, similarly for value